esmvaltool/config-references.yml

```diff
@@ -716,10 +717,11 @@ references:
   niwa-bs: "Bodeker et al., Atmos. Chem. Phys., doi:10.5194/acp-5-2603-2005, 2005."
   nsidc-0116: "Tschudi, M., W. N. Meier, J. S. Stewart, C. Fowler, and J. Maslanik. 2019. Polar Pathfinder Daily 25 km EASE-Grid Sea Ice Motion Vectors, Version 4. [Indicate subset used]. Boulder, Colorado USA. NASA National Snow and Ice Data Center Distributed Active Archive Center. doi: https://doi.org/10.5067/INAWUWO7QH7B. [Date Accessed]."
```

Comment:
Should this be adjusted as well? To "Tschudi et al., ..."

Reply:
Good catch! Fixed.